pynvjitlink/_nvjitlinklib.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+# A shim layer to use new bindings from CUDA Python
+from cuda.bindings import nvjitlink
+from cuda.bindings.nvjitlink import nvJitLinkError
+
+
+def nvjitlink_version():
+    return nvjitlink.version()
+
+
+def create(*options):
+    return nvjitlink.create(len(options), options)
+
+
+def destroy(handle):
+    return nvjitlink.destroy(handle)
+
+
+def add_data(handle, input_type, data, filename):
+    nvjitlink.add_data(handle, input_type, data, len(data), filename)
+
+
+def add_file(*args, **kwargs):
+    raise NotImplementedError("seems unused by pynvjitlink")
```

Comment:
If unimplemented, should this shim be omitted?  Is it better to get an `AttributeError` if someone tries to access this or a `NotImplementedError` when they do?

Reply:
It's been a while. Looks like I did a faithful translation 😂 
https://github.com/rapidsai/pynvjitlink/blob/666375a8696c260b1431f5292e4413516af8e706/pynvjitlink/_nvjitlinklib.cpp#L193-L197